main.inc.php

```diff
@@ -33,6 +33,15 @@
 if(!($ost=osTicket::start()) || !($cfg = $ost->getConfig()))
 Bootstrap::croak(__('Unable to load config info from DB.').' '.__('Get technical help!'));
 
+if ($cfg && $cfg->forceHttps()
+        && !defined('HTTP_OK')
```

Comment:
I actually wonder if we need to use `HTTP_OK` flag at all! It came about because of SAAS environment that required domain...etc but in general osTicket installation we can just check `osTicket::is_cli()`

Reply:
I agree with that. I can revert this to `&& !osTicket::is_cli()` and remove the added `define('HTTP_OK', TRUE);` elsewhere.